testnet/src/main.rs

```diff
@@ -29,30 +28,47 @@ pub use self::config::{Config, ConfigFile};
 pub use self::event_dispatcher::{EventDispatcher};
 pub use self::run_loop::{neon, helium};
 
-
+use pico_args::Arguments;
 use std::env;
-
+    
 fn main() {
 
-    util::log::set_loglevel(util::log::LOG_INFO).unwrap();
+    let mut args = Arguments::from_env();
+    let subcommand = args.subcommand().unwrap().unwrap_or_default();
 
-    let argv : Vec<String> = env::args().collect();
-
-    let conf = match argv.len() {
-        n if n >= 2 => {
-            println!("Starting testnet with config {}...", argv[1]);
-            Config::from_config_file_path(&argv[1])
-        },
+    let config_file = match subcommand.as_str() {
+        "mocknet" => {
+            args.finish().unwrap();
+            ConfigFile::mocknet()
+        }
+        "helium" => {
+            args.finish().unwrap();
+            ConfigFile::helium()
+        }
+        "neon" => {
+            args.finish().unwrap();
+            ConfigFile::neon()
```

Comment:
Can these functions read in their configs from the neon/helium/mocknet toml files? Otherwise, if we change the defaults, they would need to be updated in two places.

Reply:
In a ideal world, at this point, you download a binary an you can just join the `neon` network.
If we need additional files, it makes the developer experience less straightforward.